frontend/src/components/pages/TournamentRegistrations/TournamentRegistrations.tsx

```diff
@@ -92,6 +108,14 @@ export const TournamentRegistrations = () => {
     }
   };
 
+  const registerPlayer = async (playerId: string) => {
+    await TournamentRegistrationsRestApi().postPlayersForTournament({
```

Comment:
Samo registerPlayer chyba dobrze byłoby zostawić w komponencie od rejestrowania? 
A tutaj tylko poinformowac, ze sie cos takiego stało i przeładować listę?

Reply:
trochę nie rozumiem ... przecież to jest komponent od rejestrowania - nie ma osobnego komponentu od rejestrowania samych graczy

Reply:
No ale można wrzucić do osobnego pliku w tym samym komponencie

Reply:
Mhm, ale chodzi mi o to, że ten bottomsheet moglbys wszedzie pokazac gdzie chcesz i zarejestrowac gracza. Teraz nie mozesz bo on nie spelnia swojej funkcjonalnosci. Nie rejestruje gracza, tylko mozna kliknac przycisk a gracza rejestruje komponent zapisów. Komponent formularza nie ma odpowiezialnosci, ktora miec powinien, a sama rejestracja ma wiecej odpowiedzialnosci.

Reply:
no ciężko się z tym nie zgodzić :-) 
https://github.com/nowakprojects/CodersCamp2020.Project.FullStack-Node-React.TableSoccerTournaments/pull/138/commits/df0349b6aaca1b5d671840794af98324937ed168

Teraz wygląda lepiej... nie podobają mi się dalej te callbacki wszędzie, no ale nie udało mi się z tym useReducerem, więc chyba na tę chwilę to zostaje :-|
Mergować? Czy coś jeszcze z tym formularzem jest do poprawki :-) @nowakprojects

Reply:
Merguj juz to, najwyzej zmieni sie cos potem :)